src/libData/AccountData/AccountStore.cpp

```diff
@@ -291,10 +291,12 @@ void AccountStore::RevertCommitTemp() {
 
   // Revert changed
   for (auto const entry : m_addressToAccountRevChanged) {
+    // LOG_GENERAL(INFO, "Revert changed address: " << entry.first);
     (*m_addressToAccount)[entry.first] = entry.second;
     UpdateStateTrie(entry.first, entry.second);
   }
   for (auto const entry : m_addressToAccountRevCreated) {
+    // LOG_GENERAL(INFO, "Remove created address: " << entry.first);
```

Comment:
Do you want to keep this commented-out log message?

Reply:
Yeah, can just leave them there, after the state delta being fixed may also need to enable in local test and check again